compiler/src/dotty/tools/repl/Rendering.scala

```diff
@@ -26,6 +26,32 @@ private[repl] class Rendering(parentClassLoader: Option[ClassLoader] = None):
 
   var myClassLoader: AbstractFileClassLoader = uninitialized
 
+  private var BlackWhite: AnyRef = uninitialized
+  private var BlackWhite_apply: java.lang.reflect.Method = uninitialized
+  private var FansiStr_plainText: java.lang.reflect.Method = uninitialized
+  private var pprintLoaded: Boolean = false
+  private def pprintRender(value: Any, width: Int, height: Int, initialOffset: Int)(using Context): String = {
+    val cl = classLoader()
+    if !pprintLoaded then
+      val pprintCls = Class.forName("dotty.shaded.pprint.PPrinter$BlackWhite$", false, cl)
+      BlackWhite = pprintCls.getField("MODULE$").get(null)
+      BlackWhite_apply = pprintCls.getMethod("apply",
+        classOf[Any],     // value
+        classOf[Int],     // width
+        classOf[Int],     // height
+        classOf[Int],     // indentation
+        classOf[Int],     // initialOffset
+        classOf[Boolean], // escape Unicode
+        classOf[Boolean], // show field names
+      )
```

Comment:
with methodHandle？

Reply:
this isn't well suited because method handle is only optimised if its a static final field, but this is based on a dynamic class loader